languages/csharp/reference/README.md

```diff
@@ -160,23 +160,27 @@ While C# is primarily an OO language, lots of functional concepts have been adde
-| `floating-point-numbers` | Know of the existing of the three floating point types: `double`, `float` and `decimal`. Know when to use which floating point type.                                                                                                                                                                                                                                                                                                                                                                  |
-| `numbers`                | Know of the existence of the two most commonly used number types, `int` and `double`, and understand that the former represents whole numbers, and the latter floating-point numbers. Know of basic operators such as multiplication and comparison. Know how to convert from one numeric type to another. Know what implicit and explicit conversions are.                                                                                                                                           |
-| `properties`             | Know what properties are and how they relate to fields and methods. Know what backing-field properties are. Know what auto-implemented properties are. Know what calculated properties are. Know how to use property accessors to customize visibility. Know how to define the different types of properties.                                                                                                                                                                                         |
-| `strings`                | Know of the existence of the `string` type. Know how to create a string. Know of some basic methods (like finding the index of a character in a string, or returning a part the string). Know how to do basic string formatting.                                                                                                                                                                                                                                                                      |
-| `while-loops`            | Know how to write a `while` loop.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| concept                     | interpretation                                                                                                                                                                                                                                            |
```

Comment:
Your changes are based on a slightly older version of the README. It might be best to copy the [latest version](https://github.com/exercism/v3/blob/master/languages/csharp/reference/README.md) into this file and re-apply the changes.

Reply:
I'll have to edit this again since we reduced the prerequisites in the design.md to `strings` and `conditionals`.

Reply:
Yep!